cmd/nirvana/project/init.go

```diff
@@ -226,13 +234,31 @@ func (o *initOptions) templates(proj string) map[string]string {
 		"pkg/apis/api.go":                        o.templateAPI(),
 		"pkg/message/message.go":                 o.templateMessage(),
 		"pkg/version/version.go":                 o.templateVersion(),
-		"Gopkg.toml":                             o.templateGopkg(),
+		pkgConfPath:                              templateGopkg(),
 		"Makefile":                               o.templateMakefile(),
 		project.DefaultProjectFileName:           o.templateProject(),
 		"README.md":                              o.templateReadme(),
 	}
 }
 
+func (o *initOptions) exTemplateGopkg() (string, func() string) {
```

Comment:
`exTemplateGopkg()` selects correct template for package manager. So `templatePackageManager()` may be a better method name.